resources/test_fonts/ttx/names_only.ttx

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ttFont sfntVersion="\x00\x01\x00\x00" ttLibVersion="4.37">
+  <name>
+    <namerecord nameID="1" platformID="3" platEncID="1" langID="0x409">
```

Comment:
The Skia internal implementation seems to support at least MacRoman for platformID=Macintosh. What happens if that occurs in a name table with this impl?

Reply:
I found https://github.com/googlefonts/fontations/blob/main/read-fonts/src/tables/name.rs#L185 - so do I understand it right that the name table support in read_fonts handles the decoding for retrieving `name_string()` and does support `MacRoman`?

Reply:
Yes, we do support Mac Roman and the actual decoding occurs in the `read-fonts` name table implementation. I’ll change the platform id for the English subfamily name in the names font so that it’s represented in the test.